src/telemetry.ts

```diff
@@ -91,5 +98,11 @@ function createSentryInstance(enabled: boolean, integration: string) {
 }
 
 export function traceStep<T>(step: string, callback: () => T): T {
+  updateProgress(step);
   return startSpan({ name: step, op: 'wizard.step' }, () => callback());
 }
+
+let stepCounter = -1;
+export function updateProgress(step: string) {
+  setTag('progress', `${++stepCounter}-${step}`);
```

Comment:
I am wondering if our stats will get funky if we introduce new steps 🤔

Reply:
Do we need the numeric value in the tag?

Reply:
I could also imagine this being problematic in case there's different paths through the wizard.

Reply:
Good point. I removed the counter.